gotohelm/transpiler.go

```diff
@@ -1684,7 +1699,7 @@ func (t *Transpiler) getStructType(typ *types.Struct) (*packages.Package, *ast.S
 		panic("unhandled")
 	}
 
-	pack := t.Package.Imports[typ.Field(0).Pkg().Path()]
+	pack := t.packages[typ.Field(0).Pkg().Path()]
```

Comment:
This is subtle but the total import graph can contain packages that aren't directly imported. e.g. `A -> B -> C`. If `typ` is from `C` then `t.Package.Imports` won't contain it.